docs/source/examples/braket_mirror_circuit.md

```diff
@@ -59,9 +65,14 @@ on_aws = aws_device.name != "DensityMatrixSimulator"
 (examples/braket_mirror_circuit/define-the-circuit)=
 ## Define the circuit
 
-We use mirror circuits to benchmark the performance of the device. Mirror circuits, introduced in https://arxiv.org/abs/2008.11294, are designed such that only one bitstring should be sampled. When run on a device, any other measured bitstrings are due to noise. The frequency of the correct bitstring is our target metric.
+We use mirror circuits to benchmark the performance of the device. Mirror circuits, introduced in <https://arxiv.org/abs/2008.11294>, are designed such that only one bitstring should be sampled. When run on a device, any other measured bitstrings are due to noise. The frequency of the correct bitstring is our target metric.
```

Comment:
```suggestion
We use mirror circuits to benchmark the performance of the device. Mirror circuits, introduced in {cite}`Proctor_2021_NatPhys`, are designed such that only one bitstring should be sampled. When run on a device, any other measured bitstrings are due to noise. The frequency of the correct bitstring is our target metric.
```

Reply:
Done in https://github.com/unitaryfund/mitiq/pull/2008/commits/1caa9a72880c82dcbae5c6f497bf199decbaa74f